Dnn.AdminExperience/EditBar/Dnn.EditBar.UI/Services/CommonController.cs

```diff
@@ -21,27 +21,40 @@ namespace Dnn.EditBar.UI.Services
     using DotNetNuke.Entities.Tabs;
     using DotNetNuke.Security;
     using DotNetNuke.Security.Permissions;
+    using DotNetNuke.Services.Personalization;
     using DotNetNuke.Web.Api;
     using DotNetNuke.Web.Api.Internal;
-
+
     [DnnPageEditor]
     public class CommonController : DnnApiController
     {
+        private readonly PersonalizationController personalizationController;
+
+        /// <summary>Initializes a new instance of the <see cref="CommonController"/> class.</summary>
+        public CommonController()
+            : this(null)
+        {
+        }
+
+        /// <summary>Initializes a new instance of the <see cref="CommonController"/> class.</summary>
+        /// <param name="personalizationController">The personalization controller.</param>
+        public CommonController(PersonalizationController personalizationController)
+        {
+            this.personalizationController = personalizationController;
+        }
```

Comment:
Do we want to have a fallback here if the class is instantiated with the parameterless constructor?

Reply:
Yes, I'll add that